test/machine_install.py

```diff
@@ -169,7 +169,7 @@ def start(self):
             # Live install ISO does not have sshd service enabled by default
             # so we can't run any Machine.* methods on it.
             if not self.is_live():
-                Machine.wait_boot(self)
+                Machine.wait_boot(self, timeout_sec=300)
```

Comment:
It's necessary because of the one-minute grub menu timeout -- so that's just sitting around idly. Update the commit message for that.

Reply:
done.